python/cudf/cudf/_fuzz_testing/tests/fuzz_test_orc.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+import io
+import sys
+
+import cudf
+from cudf._fuzz_testing.main import pythonfuzz
+from cudf._fuzz_testing.orc import OrcReader, OrcWriter
+from cudf._fuzz_testing.utils import (
+    ALL_POSSIBLE_VALUES,
+    compare_dataframe,
+    run_test,
+)
+
+
+@pythonfuzz(
+    data_handle=OrcReader,
+    params={
```

Comment:
I think you can also vary `use_index` without braking the  output compatibility with the reference.

Reply:
Based on offline discussion - stripe selection can be used here too.

Reply:
@vuule, added the stripe selection too by validating our output with `pyorc`'s output when reading through stripes.
Added `use_index` validation aswell.